lib/trackler/problem.rb

```diff
@@ -7,81 +7,108 @@ class Problem
     def initialize(slug, root)
       @slug = slug
       @root = root
+      @xcommon_repository_url = 'https://github.com/exercism/x-common/blob/master'
     end
 
     def exists?
-      !!md && !!yaml
+      description_exists? && yaml_exists?
```

Comment:
I don't understand why the styleguide suggests avoiding the use of `!!`.

> Do you not think that description_exists? is more meaningful than !!md?

Not really, though the name `md` is not good. `!!description` would be better, to my mind.